samples/kubernetes/end2end/docs/03-wls-operator.md

```diff
@@ -82,6 +83,11 @@ kubectl get crd domains.weblogic.oracle
 NAME                      CREATED AT
 domains.weblogic.oracle   2019-05-28T07:17:26Z
 ```
+Verify the domain CRD version is correct.
+```
+kubectl get crd domains.weblogic.oracle -o jsonpath="{.spec.version}"
+```
+The expected version should be `v5`.
```

Comment:
Question: Why is the version important?  Does this sample not work with older versions of the Operator?

Reply:
Good question.

This is the domain CRD version, not a specific domain resource version. Each operator has a fix version of domain CRD. 

Since the operator is backward compatible, a domain resource with older version than the version of domain CRD should be supported. For example, domain CRD `v5` can support domain resource`v4` and `v3`. But domain CRD `v3` can not support omain resource`v4`.

This check is to make sure that no leftover domain CRD. This can happen when the wls operator is updated to a newer version but the domain CRD is not.